risc0/zkvm/platform/Cargo.toml

```diff
@@ -11,12 +11,22 @@ repository = { workspace = true }
 # used as many places as possible to share the platform definitions.
 [target.'cfg(target_os = "zkvm")'.dependencies]
 bytemuck = { version = "1.13", optional = true }
+critical-section = { version = "1.1.2", optional = true }
```

Comment:
As we add more optional crate features, I think it might be worth running `cargo clippy` in CI over every combination of features (or at least every individual feature) to ensure we don't accidentally ship a breaking change

Reply:
There seems to be a `clippy.py` script run in CI with some default feature flags, but given a feature it expects it to be present across all packages, so I successfully ran `cargo clippy -p risc0-zkvm -F heap-embedded-alloc` (and for `risc0-zkvm-platform`) manually for now.